config.py

```diff
@@ -8,6 +8,7 @@
 
 
 class Config:
+    FLASK_APP = "flaskr"
```

Comment:
I guess your app is not named `flaskr`, same to other files with `flaskr` name or `flaskr` in text

Reply:
fixed